serializability-checker-compiler-plugin/src/test/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponentSpec.scala

```diff
@@ -13,11 +14,13 @@ class SerializabilityCheckerCompilerPluginComponentSpec extends AnyWordSpecLike
   private val serYesCode = getResourceAsString("MySerializableYes.scala")
   private val serNoCode = getResourceAsString("MySerializableNo.scala")
 
-  private def testCode(resourceName: String, errorTypes: ClassType = ClassType.Message, detectionType: Int = 0) = {
-    import SerializabilityCheckerCompilerPlugin.Flags._
+  private def testCode(
+      resourceName: String,
+      errorTypes: ClassType = ClassType.Message,
+      disableFlag: String = disableGenerics) = {
```

Comment:
One extra thing! It's not clear whether _disable flag_ here means:
1. pass a flag that disables certain thing
OR
2. disable a flag (so that it's not enabled)

I suppose its (1.)... if so, it'd be better to just rename it to sth like `passFlag`

Reply:
Well... in fact this would be
3. disable a disable flag (...) so that the plugin will enable detection for  "Generics" or other chosen type of detection...

Let me change this slightly to make it more readable